lms/views/dashboard/api/grading.py

```diff
@@ -44,43 +37,55 @@ def __init__(self, request) -> None:
         permission=Permissions.GRADE_ASSIGNMENT,
         schema=AutoGradeSyncSchema,
     )
-    def auto_grading_sync(self):
+    def create_auto_grading_sync(self):
         assignment = self.dashboard_service.get_request_assignment(self.request)
-        assert assignment.lis_outcome_service_url, "Assignment without grading URL"
-        lti_registration = self.db.scalars(
-            select(LTIRegistration)
-            .join(ApplicationInstance)
-            .join(Grouping)
-            .where(Grouping.id == assignment.course_id)
-            .order_by(LTIRegistration.updated.desc())
-        ).first()
-        assert lti_registration, "No LTI registraion for LTI1.3 assignment"
 
-        sync_h_user_ids = [g["h_userid"] for g in self.request.parsed_params["grades"]]
+        if self.db.scalar(
+            select(GradingSync).where(
+                GradingSync.assignment_id == assignment.id,
+                GradingSync.status.in_(["scheduled", "in_progress"]),
+            )
+        ):
+            return 409
```

Comment:
Up to us, we could return some basic "grade already in progress"

Reply:
If we have some standard format for errors, then let's return the most basic one indicating the reason of the error. Otherwise, an empty body should work for now.